README.md

```diff
@@ -65,6 +65,7 @@ unflatten({
 ### delimiter
 
 Use a custom delimiter for (un)flattening your objects, instead of `.`.
+To use bracket notation set the delimiter option to `[]`,`{}`, or `()`.
```

Comment:
Perhaps mention any two characters can be used. I do worry a bit about the inability to escape any delimited character, though this isn't necessarily specifically a problem for your proposed change.